src/main/proto/Report.proto

```diff
@@ -60,4 +65,24 @@ message Schedule {
 message ReportLog {
     string task_status = 1;
     string task_executed = 2;
+    string threshold_met = 3;
+    string notification_sent = 4;
```

Comment:
could you please add an example what is a possible value for this? notification sent sounds like a boolean.

Reply:
yes its boolean I will correct it

Reply:
and then `threshold_met` as well should be bool?